person.rb

```diff
@@ -1,16 +1,13 @@
-require_relative 'nameable'
-require_relative 'trimmer_decorator'
-require_relative 'capitalize_decorator'
-
-class Person < Nameable
-  attr_accessor :id, :name, :age
+class Person
+  attr_accessor :id, :name, :age, :rentals
 
   def initialize(age:, name: 'Unknown', parent_permission: true)
     super()
     @id = rand(1..1000)
     @age = age
     @name = name
     @parent_permission = parent_permission
+    @rentals = []
   end
 
```

Comment:
- [ ] It would be better if our Person class has a method for adding rentals in order to achieve many-to-many relationship. Here is a [link](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/articles/oop_relationships_by_examples.md#many-to-many-relationship) to help us understand it better0
```
def add_rental(book, date)
    Rental.new(date, book, self)
end
```

Reply:
Fixed.